File: lib/ios/native/PickerViewManager.m
//
//  PickerViewManager.m
//  horizontalPicker
//
//  Created by v.blokhin on 2/13/20.
//  Copyright © 2020 Facebook. All rights reserved.
//

#import <Foundation/Foundation.h>

#import "React/RCTViewManager.h"
@interface RCT_EXTERN_MODULE(PickerViewManager, RCTViewManager)


RCT_EXPORT_VIEW_PROPERTY(items, NSArray *)
RCT_EXPORT_VIEW_PROPERTY(onItemChange, RCTDirectEventBlock)

@end

File: ios/native/PickerViewManager.swift
//
//  PickerViewManager.swift
//  horizontalPicker
//
//  Created by Valentin Blokhin on 12.02.2020.
//  Copyright © 2020 Facebook. All rights reserved.
//

@objc(PickerViewManager)
class PickerViewManager: RCTViewManager {
  override func view() -> UIView! {
    let pickerView = PickerView()
    return pickerView
  }
}
